syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetContextValueIdRequest {
    string context_value_name = 1;
}

message GetContextValueIdResponse {
    uint32 context_id = 1;
    uint32 value_id = 2;
    jio.brain.proto.knowledge.healthcare.base.Status status = 3;
}